src/mapDataManipulation/const.ts

```diff
@@ -9,10 +9,12 @@ export type Effects = {
   redRange?: ColorRange;
   greenRange?: ColorRange;
   blueRange?: ColorRange;
+  customEffect?: Function;
 };
 
-export type RgbMappingArrays = {
-  red: number[];
-  green: number[];
-  blue: number[];
+export type ImageProperties = {
+  imageData: Uint8ClampedArray;
+  imageWidth: number;
+  imageHeight: number;
+  imageFormat: string;
```

Comment:
Nitpick regarding naming: 
- `image` is probably not needed as this is `ImageProperties`
- `imageData` reminds of [ImageData](https://developer.mozilla.org/en-US/docs/Web/API/ImageData), which is a different structure - maybe `rgbaArray` or just `rgba` would be a better name?

Fftr either way, not very important.

Reply:
Thanks, changed anyway.